### PR TITLE
Sanity pass on change notification topics

### PR DESCRIPTION
**Pull request by jasonjoh**
I went over the change notification topics for accuracy and consistency. This effort is to enable us to retire the [Outlook-specific topic](https://docs.microsoft.com/en-us/outlook/rest/webhooks) and just redirect to the Graph topic.

- Revised and reorganized the conceptual topic
- Cut a lot of content out of the reference topics
- Changed titles on conceptual and reference per discussion with Angela
- Updated subscription resource topics to indicate which fields are required or optional

### CODE REVIEW

File: api-reference/beta/resources/webhooks.md [hunk 1]
```diff
@@ -1,198 +1,43 @@
...
+
+- Messages
+- Events
+- Contacts
+- Users
+- Groups
+- Group conversations
+- Content shared on OneDrive including drives associated with SharePoint sites
+- User's personal OneDrive folders
+
```
Comment (Lauragra) [inline api-reference/beta/resources/webhooks.md]: Add a comma before including, and change User's to Users'.

File: api-reference/beta/resources/webhooks.md [hunk 2]
```diff
@@ -1,198 +1,43 @@
...
+## Permissions
+
+In general, subscription operations require read permission to the resource. For example, to get notifications for messages, your app needs the `Mail.Read` permission. The [create subscription](../api/subscription_post_subscriptions.md) article lists permissions needed for each resource type. The following table lists the types of permissions your app can request to use webhooks for specific resource types.
+
```
Comment (Lauragra) [inline api-reference/beta/resources/webhooks.md]: We don't apply the inline code style to permission names, FYI.

File: api-reference/v1.0/resources/subscription.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
-# Subscription Resource Type
-A subscription allows a client app to receive notifications about changes to data on the Microsoft Graph. Currently, subscriptions are enabled for the following datasets:
+# subscription resource type
 
-1. Mail, events, and contacts from Outlook.
-1. Conversations from Office Groups.
-1. Drive root items from OneDrive.
-1. Users and Groups from Azure Active Directory.
+A subscription allows a client app to receive notifications about changes to data on the Microsoft Graph. Currently, subscriptions are enabled for the following resources:
```
Comment (Lauragra) [inline api-reference/v1.0/resources/subscription.md]: Change "on the Microsoft Graph" to "in Microsoft Graph".

File: api-reference/v1.0/resources/webhooks.md [hunk 4]
```diff
@@ -1,214 +1,35 @@
-# Working with Webhooks in Microsoft Graph
+# Use the Microsoft Graph API to get change notifications
 
-The Microsoft Graph REST API uses a webhook mechanism to deliver notifications to clients. A client is a web service that configures its own URL to receive notifications. Client apps use notifications to update their state upon changes.
+The Microsoft Graph REST API uses a webhook mechanism to deliver notifications to clients. A client is a web service that configures its own URL to receive notifications. Client apps use notifications to update their state upon changes. For more details, including how to subscribe for and handle incoming notifications, see [Set up notifications for changes in user data](../../../concepts/webhooks.md).
```
Comment (Lauragra) [inline api-reference/v1.0/resources/webhooks.md]: Global: same edits apply to the v1.0 topic.

File: api-reference/beta/resources/webhooks.md (hunk 2 above)
Comment (jasonjoh) [reply api-reference/beta/resources/webhooks.md]: It was there before I edited the doc. Why don't we do that?

File: concepts/webhooks.md [hunk 5]
```diff
@@ -22,69 +26,75 @@ Or a specific ID:
...
-Or on a Sharepoint / OneDrive for Business drive:
+Or on a SharePoint / OneDrive for Business drive:
 `/drive/root`
 
 Or on a user's personal OneDrive:
 `/drives/{id}/root`
 `/drives/{id}/root/subfolder`
 
-After Microsoft Graph accepts the subscription request, it pushes notifications to the URL specified in the subscription. The app then takes action according to its business logic. For example, it fetches more data, updates cache and views, etc.
+### Azure AD Resource Limitations
```
Comment (Lauragra) [inline concepts/webhooks.md]: Change to sentence caps - Azure AD resource limitations.

File: concepts/webhooks.md [hunk 6]
```diff
@@ -22,69 +26,75 @@ Or a specific ID:
...
 
-Apps need to renew their subscriptions before the expiration time. Otherwise, they need to create a new subscription. For a list of maximum expiration times, see [Maximum length of subscription per resource type](../api-reference/v1.0/resources/subscription.md#maximum-length-of-subscription-per-resource-type).
+Certain limits apply to Azure AD based resources (users, groups) and may generate errors when exceeded:
 
-Apps can also unsubscribe at any time to stop getting notifications.
+- Maximum subscription quotas:
 
-In general, subscription operations require read permission to the resource. For example, to get notifications for messages, your app needs the `Mail.Read` permission. The [create subscription](../api-reference/v1.0/api/subscription_post_subscriptions.md) article lists permissions needed for each resource type. The following table lists the types of permissions your app can request to use webhooks for specific resource types. 
+  - Per App: 50,000 total subscriptions
+  - Per Tenant: 35 total subscriptions across all apps
```
Comment (Lauragra) [inline concepts/webhooks.md]: App and Tenant should be lowercase here.

File: concepts/webhooks.md [hunk 7]
```diff
@@ -22,69 +26,75 @@ Or a specific ID:
...
+  - Per App and Tenant combination: 7 total subscriptions
 
-| Permission type                        | Supported resource types in v1.0                                 |
-| :------------------------------------- | :--------------------------------------------------------------- |
-| Delegated - work or school account     | [contact][], [conversation][], [drive][], [event][], [message][] |
-| Delegated - personal Microsoft account | None                                                             |
-| Application                            | [contact][], [conversation][], [event][], [message][]            |
+- Azure AD B2C tenants are not supported
 
+- Consumer account Users not supported
```
Comment (Lauragra) [inline concepts/webhooks.md]: Users should be lowercase; these two bullets should end with periods.

**Comment by angelgolfer-ms** [comment]
A bit late to the review request. The changes were very close to as we had discussed, and the enhancements look good to me! Thank you, Jason.

**Comment by jasonjoh** [comment]
Per new guidance, changed target of this PR to Staging branch.

**Pull request merged**